scripts/auto-rebase/rebase.sh

```diff
@@ -891,31 +1035,38 @@ rebase_to() {
 
 usage() {
     echo "Usage:"
-    echo "$(basename "$0") to RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM         Performs all the steps to rebase to a release image. Specify both amd64 and arm64."
-    echo "$(basename "$0") download RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM   Downloads the content of a release image to disk in preparation for rebasing. Specify both amd64 and arm64."
-    echo "$(basename "$0") buildfiles                                       Updates build files (Makefile, Dockerfile, .spec)"
-    echo "$(basename "$0") go.mod                                           Updates the go.mod file to the downloaded release"
-    echo "$(basename "$0") generated-apis                                   Regenerates OpenAPIs"
-    echo "$(basename "$0") images                                           Rebases the component images to the downloaded release"
-    echo "$(basename "$0") manifests                                        Rebases the component manifests to the downloaded release"
+    echo "$(basename "$0") to RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM LVMS_RELEASE_MULTI_ARCH        Performs all the steps to rebase to a release image. Specify both amd64 and arm64 ocp releases, and multi-arch lvms operator bundle image."
+    echo "$(basename "$0") download RELEASE_IMAGE_INTEL RELEASE_IMAGE_ARM LVMS_RELEASE_MULTI_ARCH  Downloads the content of a release image to disk in preparation for rebasing. Specify both amd64 and arm64 and multi-arch lvms operator bundle image."
+    echo "$(basename "$0") buildfiles                                                              Updates build files (Makefile, Dockerfile, .spec)"
+    echo "$(basename "$0") go.mod                                                                  Updates the go.mod file to the downloaded release"
+    echo "$(basename "$0") generated-apis                                                          Regenerates OpenAPIs"
+    echo "$(basename "$0") images                                                                  Rebases the component images to the downloaded release"
+    echo "$(basename "$0") manifests                                                               Rebases the component manifests to the downloaded release"
     exit 1
 }
 
 command=${1:-help}
 case "$command" in
     to)
-        [[ $# -ne 3 ]] && usage
-        rebase_to "$2" "$3"
+        [[ $# -ne 4 ]] && usage
+        rebase_to "$2" "$3" "$4"
         ;;
     download)
-        [[ $# -ne 3 ]] && usage
+        [[ $# -ne 4 ]] && usage
```

Comment:
Does the automated job get this new argument?

Reply:
Not yet.  To prevent breaking the job in CI, I intend to update CI after this is merged to update the env vars that are consumed by the wrapper script (rebase-script-automation.sh), and then glue it all together with an update to wrapper script to pass the env var thru. Hopefully, this will mean the rebase job doesn't see any downtime.

Reply:
This line changes expectation that script will receive one more argument, so shouldn't we change job config so we have this available in container and then merge this change + changes in `rebase_job_entrypoint.sh`?
If you merge this first and not change `rebase_job_entrypoint.sh` then the script will fail expecting 4 args when given 3 or I missed something?